state-sync/state-sync-v2/state-sync-driver/src/error.rs

```diff
@@ -44,6 +44,31 @@ pub enum Error {
     UnexpectedError(String),
 }
 
+impl Error {
+    /// Returns a summary label for the error
+    pub fn get_label(&self) -> &'static str {
```

Comment:
there's no macro for this? 🤦

Reply:
@zekun000, there must be something somewhere. I've created an issue: https://github.com/aptos-labs/aptos-core/issues/531

Reply:
I commented on that one. Looks like we have a different philosophy around dependencies now, lol.